README.md

```diff
@@ -55,6 +55,14 @@ instead of `make shell` as the latter does not bind port 8000 which you need to
 
 Run `make help` to see other helpful commands.
 
+### Compiling LESS files
+
+By default, the `.less` files do not compile in local development because it impacts load times on OSX. If you are working on `.less` files, you can enable compilation with the following environment variable:
+
+  ```
+  echo "PIPELINE_COLLECTOR_ENABLED=True" >> .env
```

Comment:
👍

Reply:
@pmac Happy to help ! I'll leave it to you to merge this in once ready (I don't have write access to the repo)

Reply:
Sounds good. I'll let @akatsoulas have the final look and let him merge it since he's primary on the project. But it all looks good to me! 🍰